src/wasm/wasm-debug.cpp

```diff
@@ -727,7 +735,8 @@ static void iterContextAndYAML(const T& contextList, U& yamlList, W func) {
 static void updateDIE(const llvm::DWARFDebugInfoEntry& DIE,
                       llvm::DWARFYAML::Entry& yamlEntry,
                       const llvm::DWARFAbbreviationDeclaration* abbrevDecl,
-                      const LocationUpdater& locationUpdater) {
+                      LocationUpdater& locationUpdater,
+                      BinaryLocation baseAddress) {
```

Comment:
perhaps `unitAddress`? (otherwise not obvious what it's the base address of, or is that something obvious in the DWARF world?)